package/yast2-users.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Thu Feb 18 09:23:10 CET 2016 - schubi@suse.de
+
+- Checking empty string while generating home directory.
```

Comment:
No, here you're describing the internals which is not helpful to the user.
Better:
> Do not crash autoinstallation if home directory is directly under /.